go/rpc/client.go

```diff
@@ -17,6 +17,7 @@ const (
 
 	Health = "ten_health"
 	Config = "ten_config"
+	RPCKey = "ten_rPCKey"
```

Comment:
Not sure if it's case sensitive anyway but this capitalisation seems ugly haha, maybe copilot did that.

Reply:
I called the method RPCKey, and the convention is `prefix_{lowerCaseFirstChar}{otherChars}`

Reply:
Fair, I'd probably rename it to `RpcKey()` to fix that if it doesn't break our linter with a comment to explain it keeps the method name nice lol. I guess that method's not super public facing but a little bit.